cmds/task/run.go

```diff
@@ -84,7 +84,7 @@ func runRunTask(cmd *cobra.Command, args []string) error {
 	}
 
 	// Generate a new taskID
-	taskID := slugid.Nice()
+	taskID := slugid.V4()
```

Comment:
As above, please leave this using `Nice`.

Reply:
I don't know how that happened. Haven't touch that ...